operating_systems/opensuse/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG BASEIMAGE=opensuse/leap
```

```suggestion
ARG BASEIMAGE=docker.io/opensuse/leap
```
Explicitly naming the registry can avoid issues with Podman in some cases (when Podman doesn't know which registry to use). In this case this is only for conformity sake though.